operators/multiclusterobservability/manifests/base/config/metrics_allowlist.yaml

```diff
@@ -227,8 +220,6 @@ data:
           - kube_pod_container_resource_limits 
           - kube_pod_container_resource_requests 
           - kube_resourcequota 
-          - mixin_pod_workload 
-          - namespace_workload_pod:kube_pod_owner:relabel 
```

Comment:
Removed recording rules from the collection rules `SNOHighMemoryUsage` list. These will still be captured within the default metrics list.